communication/mav_modes.h

```diff
@@ -222,13 +222,14 @@ typedef enum
 	CUST_DESCENT_TO_SMALL_ALTITUDE = 8,			///< First auto landing behavior
 	CUST_DESCENT_TO_GND = 16,					///< Second auto landing behavior, comes after DESCENT_TO_SMAL_ALTITUDE
 	
-	CUST_COLLISION_AVOIDANCE = 32,
-
-	CUST_BATTERY_LOW = 64,
-	CUST_FENCE_1 = 128,
-	CUST_HEARTBEAT_LOST = 256,
-	CUST_REMOTE_LOST = 512,
-	CUST_GPS_BAD = 1024
+	CUST_COLLISION_AVOIDANCE = 32,				///< Collision avoidance 
+
+	CUST_BATTERY_LOW = 64,						///< Battery low flag
+    CUST_FENCE_1 = 128,							///< Fence 1 violation flag
+    CUST_FENCE_2 = 256,							///< Fence 2 violation flag
```

Comment:
FENCE_1 and FENCE_2 are not explicit, 

Apparently they are for horizontal and vertical limits, so what about FENCE_XY and FENCE_Z, or FENCE_HORIZONTAL and FENCE_VERTICAL ?